packages/plugins/users-permissions/server/controllers/auth.js

```diff
@@ -294,34 +294,7 @@ module.exports = {
       throw new ApplicationError('Impossible to find the default role');
     }
 
-    const { email, username, provider } = params;
-
-    const identifierFilter = {
-      $or: [
-        { email: email.toLowerCase() },
-        { username: email.toLowerCase() },
-        { username },
-        { email: username },
```

Comment:
Since we're logging in by identifier instead of username or email, we're also checking uniqueness across email + username, only adding unique: true to the model won't check that.